### ISSUE TITLE
Move power_range into the parameters for zh

### ISSUE THREAD

**Issue by Jacfomg** [qiboteam/qibolab#814]
I think reading the `power_level` parameters for Zh from the `parameters.json` would be simpler for people calibrating as well as making qibo save at each report. This would follow what qm does. @hay-k could you have a look and maybe include this as part of your refactor ?

**Comment by hay-k** [comment]
This looks like pretty much independent from the refactor, so I would rather do it separately, not to clog the refactor PR too much.

**Comment by alecandido** [comment]
I agree with @hay-k, let's try to merge that PR as soon as possible.
